src/main/scripts/org/reaktivity/specification/nukleus/oauth/streams/proxy/request.with.scopes.with.signed.jwt.rs256.forwarded/accept/client.rpt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property scopes "1 2 3"
```

Comment:
Better to inline `scopes` directly below instead, otherwise consumers of the script are tempted to parameterize its usage which leads to more obscure testing.